flang/test/Driver/dependent-lib.f90

```diff
@@ -0,0 +1,7 @@
+
+! RUN: %flang_fc1 -emit-mlir -triple aarch64-pc-windows-msvc --dependent-lib=libtest %s -o - 2>&1 | FileCheck %s
+! RUN: %flang_fc1 -emit-mlir -triple x86_64-pc-windows-msvc --dependent-lib=libtest %s -o - 2>&1 | FileCheck %s
```

Comment:
[nit] If you only want to check the triple, then you could use LIT's `DEFINE`/`REDEFINE` here.

Reply:
I changed to use this but I'm not sure it's actually any clearer than just listing the run lines. Let me know what you think now that I've put it in 👍